lib/l10n/app_it.arb

```diff
@@ -0,0 +1,260 @@
+  "piped_instance": "Piped Server Instance",
+  "piped_description": "The Piped server instance to use for track matching",
+  "piped_warning": "Qualcuno di questi potrebbe non funzionare. Quindi usali a tuo rischio e pericolo",
+  "generate_playlist": "Crea Playlist",
+  "track_exists": "Il Brano {track} esiste già",
+  "replace_downloaded_tracks": "Sostituisci tutti i brani scaricati",
+  "skip_download_tracks": "Salta il download di tutti i brani scaricati",
+  "do_you_want_to_replace": "Vuoi sostituire i brani esistenti??",
+  "replace": "Sostituisci",
+  "skip": "Salta",
+  "select_up_to_count_type": "Scegli finao a {count} {type}",
+  "select_genres": "Scegli Generi",
+  "add_genres": "Aggiungi Generi",
+  "country": "Paese",
+  "number_of_tracks_generate": "Numero di brani da generare",
+  "acousticness": "Acusticità",
+  "danceability": "Ballabilità",
+  "energy": "Energia",
+  "instrumentalness": "Strumentalità",
+  "liveness": "Vivacità",
+  "loudness": "Sonorità",
+  "speechiness": "Dissolutezza",
+  "valence": "Valenza",
+  "popularity": "Popolarità",
+  "key": "Key",
+  "duration": "Durata (s)",
+  "tempo": "Tempo (BPM)",
+  "mode": "Modalità",
+  "time_signature": "Time Signature",
+  "short": "Corto",
+  "medium": "Medio",
+  "long": "Lungo",
+  "min": "Min",
+  "max": "Max",
+  "target": "Target",
+  "moderate": "Moderato",
+  "deselect_all": "Deseleziona Tutto",
+  "select_all": "Seleziona Tutto",
+  "are_you_sure": "Sei sicuro?",
+  "generating_playlist": "Sto Creando la tua playlist personalizzata...",
+  "selected_count_tracks": "{count} brani selezionati",
+  "download_warning": "Se scarichi tutti i brani contemporaneamente stai palesemente piratando la Musica e stai causando un danno alla società creativa della Musica. Spero tu ne sia a conoscienza. Cerca sempre di rispettare e supportare il duro lavoro dell'Artista",
+  "download_ip_ban_warning": "Comunque, il tuo IP potrebbe essere bloccato da YouTube per via dell'elevato numero di richieste di download. Il blocco dell'IP significa che non potrai usare Youtube (anche se hai effettuato l'accesso) per almeno 2-3 mesi da quel dispositivo IP. Spotube non si assume nessuna responsabilità se questo dovesse accadere.",
+  "by_clicking_accept_terms": "Cliccando su 'accetta' acconsentirai ai seguenti termini e condizioni:",
+  "download_agreement_1": "Sono consapevole che sto piratando la muscia. Sono un tacchino 🦃",
+  "download_agreement_2": "Supporterò l'Artista appena potrò e sto facendo questo solo perchè non ho i soldi per poter comprare la sua musica",
+  "download_agreement_3": "Sono completamente consapevole che il mio IP può essere bloccato su YouTube e non ritengo responsabile Spotube o i suoi proprietari/collaboratori per qualsiasi incidente causato dalla mia attuale azione",
+  "decline": "Rifiuta",
+  "accept": "Accetta",
+  "details": "Dettagli",
+  "youtube": "YouTube",
+  "channel": "Canale",
+  "likes": "Likes",
+  "dislikes": "Dislikes",
```

Comment:
May is better if we use "Mi Piace" and "Non Mi piace". I italian is common to use "like" in reference to social post but dislike is not used so much.

Reply:
Fixed in 940ff32